src/map.cpp

```diff
@@ -6726,13 +6727,38 @@ void map::remove_rotten_items( Container &items, const tripoint &pnt )
     const tripoint abs_pnt = getabs( pnt );
     for( auto it = items.begin(); it != items.end(); ) {
         if( has_rotten_away( *it, abs_pnt ) ) {
+            //If the item that is rotting is a food item, see if we can spawn a monster
```

Comment:
Why do you feel the need to repeat the code as comment. "if the item ... is a food item" is the same as `if(it->is_comestible()) {`.